src/js/stickyKeysAssessment.js

```diff
@@ -0,0 +1,46 @@
+/*
+Copyright 2015 OCAD University
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
+*/
+
+(function ($, fluid) {
+
+    "use strict";
+
+    fluid.registerNamespace("gpii.firstDiscovery.keyboard");
+
+    fluid.defaults("gpii.firstDiscovery.keyboard.stickyKeysAssessment", {
+        gradeNames: ["fluid.modelRelayComponent", "autoInit"],
+        model: {
+            input: "",
+            offerAssistance: false
+        },
+        modelRelay: {
+            source: "input",
+            target: "offerAssistance",
+            barckward: "never",
+            forward: "liveOnly",
```

Comment:
As far as I know, the scope for the model relay should be kept as open as we can. The `backward` is not necessary since "fluid.transforms.free" doesn't provide backward transformation anyway. The `forward` also can be eliminated if `model.input` is initialized to `undefined`.